onnxruntime/contrib_ops/cuda/decoder/fastertransformer_decoder_attention/decoder_masked_multihead_attention_impl.h

```diff
@@ -10,9 +10,12 @@ namespace onnxruntime {
 namespace contrib {
 namespace cuda {
 
-struct DecoderMaskedSelfAttentionParams : AttentionParameters {
+struct DecoderMaskedMultiHeadAttentionParams : AttentionParameters {
   int beam_width = 1;
 
+  bool is_mha = false;
```

Comment:
Maybe add some more comments to describe the parameter - `is_mha` ? It seems like it is a flag to differentiate the "combined" kernel (Self + Cross) from the kernel that only does Self ?